about.html

```diff
@@ -0,0 +1,80 @@
+        <li><a href="contact.html">Contact</a></li>
+      </ul>
+    </Nav>
+  </header>
+  <main>
+    <div>
+        <h2> Education
+          </h2>
+          <p>
+              Cat ipsum dolor sit amet, lay on arms while you're using the keyboard howl uncontrollably for no reason 
+              stand in front of the computer screen give me some of your food give me some of your food give me some of your food meh, 
+              i don't want it. Nyan fluffness ahh cucumber! ooooh feather moving feather! yet poop in the plant pot, yet when owners are 
+              asleep, cry for no apparent reason but prance along on top of the garden fence, annoy the neighbor's dog and make it bark man 
+              running from cops stops to pet cats, goes to jail i like to spend my days sleeping and eating fishes that my human fished for 
+              me we live on a luxurious yacht, sailing proudly under the sun, i like to walk on the deck, watching the horizon, dreaming of 
+              a good bowl of milk.
+      
+          </p> 
+    </div>
+    <div>
+            <h2> Experience
+              </h2>
+              <p>
+                  Spill litter box, scratch at owner, destroy all furniture, especially couch cat snacks, i could pee on 
+                  this if i had the energy slap owner's face at 5am until human fills food dish. Sleep nap roll on the floor 
+                  purring your whiskers off or chase laser do not try to mix old food with new one to fool me! no, you can't 
+                  close the door, i haven't decided whether or not i wanna go out. Curl up and sleep on the freshly laundered 
+                  towels refuse to leave cardboard box, or spill litter box, scratch at owner, destroy all furniture, especially 
+                  couch.     
+              </p> 
+        </div>
+        <div>
+                <h2> Personal
+                  </h2>
+                  <p>
+                      My slave human didn't give me any food so i pooped on the floor ask to go outside and ask to come inside 
+                      and ask to go outside and ask to come inside. Step on your keyboard while you're gaming and then turn in 
+                      a circle annoy kitten brother with poking annoy owner until he gives you food say meow repeatedly until 
+                      belly rubs, feels good and poop in litter box, scratch the walls for cats go for world domination. Pet me 
+                      pet me pet me pet me, bite, scratch, why are you petting me stare out the window.     
+              
+                  </p> 
+            </div>   
+    <div>
+            <img src="images/csrlp-ppl.jpg" alt="Summer lunch program activity at PPL">
+    </div> 
+    <div>
+            <img src="images/minecraft-club-ppl.jpg" alt="Minecraft Club at PPL">
+    </div>                 
+  </main>
+  <footer>
+      <div> 
+        <p>&copy; 2019 Stephanie L Lawson</p>
+      </div>
+      <div> 
+        <button>
```

Comment:
buttons are for actions. this should be a link to your resume.

Reply:
I will include an older resume as I am currently updating.